proto/proto/chain.proto

```diff
@@ -11,6 +11,10 @@ message ChainParams {
   uint64 epoch_duration = 2;
   // The number of epochs an unbonding note for before being released.
   uint64 unbonding_epochs = 3;
+  // The maximum number of validators in the consensus set.
+  uint64 validator_limit = 4;
+  // The penalty to be applied to slashed validators' rates.
+  uint64 slashing_penalty = 5;
```

Comment:
What are the units here? Percent? Basis points?

Reply:
Currently units but this was a "good enough for the moment" decision. Percentage might be better, I was hoping for feedback from the team.